crmsh/sbd.py

```diff
@@ -184,8 +218,7 @@ def _determine_sbd_watchdog_timeout(self):
         if utils.is_qdevice_configured() and utils.service_is_active("corosync-qdevice.service"):
             qdevice_sync_timeout = utils.get_qdevice_sync_timeout()
             self._sbd_watchdog_timeout = qdevice_sync_timeout + 5
-            if self._is_s390 and self._sbd_watchdog_timeout < 15:
-                self._sbd_watchdog_timeout = 15
+            self._correct_sbd_watchdog_timeout_for_s390()
             self._stonith_timeout = self.calculate_stonith_timeout(self._sbd_watchdog_timeout)
         # add sbd and qdevice together from beginning
         elif self._context.qdevice_inst:
```

Comment:
BUG: my reading is the following wrongly reset `self._sbd_watchdog_timeout` to crmsh's hard-coded defaults. It could have been loaded from profiles.yml in _initialize_sbd(), earlier.

Reply:
Changed as
```Python
-            self._sbd_watchdog_timeout = self.SBD_WATCHDOG_TIMEOUT_DEFAULT_WITH_QDEVICE
+            if self._sbd_watchdog_timeout < self.SBD_WATCHDOG_TIMEOUT_DEFAULT_WITH_QDEVICE:
+                bootstrap.warn("sbd_watchdog_timeout is set to {} for qdevice, it was {}".format(self.SBD_WATCHDOG_TIMEOUT_DEFAULT_WITH_QDEVICE, self._sbd_watchdog_timeout))
+                self._sbd_watchdog_timeout = self.SBD_WATCHDOG_TIMEOUT_DEFAULT_WITH_QDEVICE
```